docs/serving/samples/hello-world/helloworld-java-spring/README.md

```diff
@@ -1,32 +1,29 @@
-A simple web app written in Java using Spring Boot 2.0 that you can use for
-testing. It reads in an env variable `TARGET` and prints "Hello \${TARGET}!". If
-TARGET is not specified, it will use "World" as the TARGET.
+# Hello World - Spring Boot Java
```

Comment:
Can you update the name of this in the left nav so it matches the page title?
https://deploy-preview-3728--dev-knative.netlify.app/development/serving/samples/hello-world/helloworld-java-spring/

The left nav has "Java (Spring)" so it's a bit confusing / not clear it's Spring Boot to me?

Reply:
I am regretting touching this file at all, we're supposed to be doing samples in Late beta